messages/cache_setting/create.go

```diff
@@ -0,0 +1,31 @@
+package cachesetting
+
+const (
+	CreateUsage                          = "cache-setting"
+	CreateShortDescription               = "Creates a new Cache Setting configuration"
+	CreateLongDescription                = "Creates a Cache Settings configuration based on given attributes to be used in edge applications"
+	CreateFlagEdgeApplicationId          = "Unique identifier for an edge application"
+	CreateFlagName                       = "The Cache Settings configuration name"
+	CreateFlagIn                         = "Path to a JSON file containing the attributes of the Cache Settings configuration that will be created; you can use - for reading from stdin"
+	CreateOutputSuccess                  = "Created Cache Settings configuration with ID %d\n"
+	CreateHelpFlag                       = "Displays more information about the create subcommand"
+	CreateFlagBrowserCacheSettings       = "Configures the amount of time that the content is cached in the web browser"
+	CreateFlagQueryStringFields          = "Gives a list of query strings parameters to be considered in the Cache Settings configuration, that will segregate the cache to the same URL"
+	CreateFlagCookieNames                = "Distinguishes objects in the Azion cache by name/value of cookies"
+	CreateFlagCacheByCookies             = "Whether cache by cookies is active or not"
+	CreateFlagCacheByQueryString         = "Defines how you want the content to be cached according to variations of Query String in your URLs"
+	CreateFlagCdnCacheSettings           = "Configures the amount of time Azion's Edge Applications take to cache the content. It can either Honor Origin Cache Headers or Override Cache Settings"
+	CreateFlagCachingForOptions          = "Whether caching for options is active or not"
+	CreateFlagCachingStringSort          = "Whether caching string sort is active or not"
+	CreateFlagCachingForPost             = "Whether caching for post is active or not"
```

Comment:
This command and the previous options shouldn't have the `Enabled` also, like the slice commando below? 
like:
```suggestion
	CreateFlagCachingForPostEnabled            = "Whether caching for post is active or not"
```